modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -172,7 +172,7 @@ def build_claim_structure(data:, lighthouse_id:, upstream_id:) # rubocop:disable
             claim_type_code: data[:bnft_claim_type_cd],
             claim_type: data[:claim_status_type],
             close_date: data[:claim_complete_dt].present? ? format_bgs_date(data[:claim_complete_dt]) : nil,
-            contention_list: data[:contentions]&.split(',')&.collect(&:strip) || [],
+            contention_list: data[:contentions]&.split(/(?<=\)),/)&.collect(&:strip) || [],
```

Comment:
the regex in the ticket is slightly different. is there a reason for the change? 

for reference: `"(?<=\\)),"`

just has an additional `\`.  I imagine we didn't need to escape that `\` the same as they did because of the way you have it set up? Just confirming. If so, good to merge.

Reply:
yea, I think it's a diff in languages, the double `\\` is needed depending on the language (I.E. if it's in a `""` string). In this particular context, if I did `\\`, then the contentions would have to be `name (New\)`.